src/theory/uf/ho_extension.cpp

```diff
@@ -316,7 +316,8 @@ unsigned HoExtension::checkAppCompletion()
             // add to pending list
             apply_uf[rop].push_back(n);
           }
-          // arguments are also relevant operators  FIXME (github issue #1115)
+          // Arguments are also relevant operators.
+          // It might be possible include fewer terms here, see #1115.
```

Comment:
Since you're gonna close the issue shouldn't you remove mention to it?

Reply:
No, unless there is a FIXME or TODO, we can reference issues in comments. This is helpful for remembering what the issue was.